swap-create@.service

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Create swap on /dev/%i
+Wants=systemd-modules-load.service
+After=systemd-modules-load.service
+After=dev-%i.device
+DefaultDependencies=false
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=-modprobe zram
+ExecStart={generator} --setup-device '%i'
```

Comment:
I think the template should just say `/usr/lib/systemd/system-generators/zram-generator`. That's the only place where the generator is supposed to go (as long as split-usr systems are ignored and a few other details ;)).